examples/3-apps/1-contact-list/README.md

```diff
@@ -0,0 +1,78 @@
+
+# Jitar | Contact list
+
+This example contact list application is build with build with React, MongoDB and Jitar (ReMoJi stack).
+It includes frontend and backend components and a database for storing contacts.
+
+If you want to create a new application with the same setup, you can use the following command.
+
+```
+npm create jitar@latest
+```
+
+Then type the name of the project and select the frontend framework you want.
+
+## Running the example
+
+For running the example, Docker is required for setting up MongoDB.
+
+1 Install all dependencies by running the following command from the root directory of the example.
```

Comment:
I think there should be a dot after the number, like this `1. Install ...`

Reply:
I agree, but it will intend the line because markdown will tread it as a list. I've also tried to use a `)` but that has the same result.

Reply:
I've added an escaped . after the numbers